controller/work_item_link_type.go

```diff
@@ -89,8 +91,7 @@ func enrichLinkTypeList(ctx *workItemLinkContext, list *app.WorkItemLinkTypeList
 	for categoryID := range categoryIDMap {
 		modelCategory, err := ctx.Application.WorkItemLinkCategories().Load(ctx.Context, categoryID)
 		if err != nil {
-			jerrors, httpStatusCode := jsonapi.ErrorToJSONAPIErrors(err)
-			return ctx.ResponseData.Service.Send(ctx.Context, httpStatusCode, jerrors)
+			err
```

Comment:
if you `return err` then verify that the error is properly captured and returned in the places where this function is being called. That is also why I mentioned it for workitemtype.go controller.
``
err = enrichLinkTypeList(linkCtx, &appLinkTypes)
if err != nil {
	return jsonapi.JSONErrorResponse(ctx, goa.ErrInternal("Failed to enrich link types: %s", err.Error()))
}
```